client/src/components/Components.tsx

```diff
@@ -20,8 +22,10 @@ export function TextInputField({
       setState(e.target.value);
     }
   };
+
   return (
     <Input
+      id={elementId ? elementId : ""}
```

Comment:
If someone doesn't set id for multiple TextInputs, your html will have Input tags with multiple id="" - which is not intended. 

Option 1:  make elementId compulsory not optional
Option 2: remove elementId if its just being used for testing, and find by another paramater

Reply:
Changed `""` to `undefined` which react drops from rendering

Reply:
SGTM